src/main/java/org/fenixedu/academic/domain/serviceRequests/AcademicServiceRequest.java

```diff
@@ -264,26 +261,6 @@ public String getDescription() {
         return getDescription(getAcademicServiceRequestType());
     }
 
-    @Atomic
-    final public void process() throws DomainException {
```

Comment:
it's probably best we have this merged and let the pipeline tell us if we're good to go. These seem to no longer be used anywhere but it's kinda hard to be 100 % sure

Reply:
@joao-gundersen && @joaojtmarques-qubit since we can't really be sure due to the nature of usages in JSPs I suggest that until we totally descontinue this service requests we keep this code.
Let's just send a signal in the send email and then we treat that in base module

Reply:
> Let's just send a signal in the send email and then we treat that in the customers ulisboa module

I'm afraid I don't really understand this. Is this referring to the sendConcludeMail() method? And what would we "treat" in the costumers ulisboa module?